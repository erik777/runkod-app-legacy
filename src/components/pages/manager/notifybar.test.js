import React from 'react';
import NotifyBar from './notifybar';
import TestRenderer from 'react-test-renderer';


const fnProps = {
  invalidateUiFlag: () => {

  }
};

test('1- show example project notification', () => {
  const props = Object.assign({}, {
    ui: {
      epFlag: true
    }
  }, fnProps);

  const renderer = TestRenderer.create(
    <NotifyBar {...props}/>
  );

  expect(renderer.toJSON()).toMatchSnapshot();
});

test('2- should render null', () => {
  const props = Object.assign({}, {
    ui: {
      epFlag: false
    }
  }, fnProps);

  const renderer = TestRenderer.create(
    <NotifyBar {...props}/>
  );

  expect(renderer.toJSON()).toMatchSnapshot();
});