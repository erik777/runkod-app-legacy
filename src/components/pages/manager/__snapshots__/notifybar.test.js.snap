// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- show example project notification 1`] = `
<div
  className="notify-bar clickable"
>
  <svg
    viewBox="0 0 512 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M256 90c44.3 0 86 17.3 117.4 48.6C404.7 170 422 211.7 422 256s-17.3 86-48.6 117.4C342 404.7 300.3 422 256 422s-86-17.3-117.4-48.6C107.3 342 90 300.3 90 256s17.3-86 48.6-117.4C170 107.3 211.7 90 256 90m0-42C141.1 48 48 141.1 48 256s93.1 208 208 208 208-93.1 208-208S370.9 48 256 48z"
    />
    <path
      d="M277 360h-42V235h42v125zm0-166h-42v-42h42v42z"
    />
  </svg>
  <span
    className="text-content"
    onClick={[Function]}
  >
     Our starter project can help you to understand how Runkod works. Click here to see instructions.
  </span>
  <div
    className="close-bar"
  >
    <span
      className="close-button"
      onClick={[Function]}
    >
      <svg
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M405 136.798L375.202 107 256 226.202 136.798 107 107 136.798 226.202 256 107 375.202 136.798 405 256 285.798 375.202 405 405 375.202 285.798 256z"
        />
      </svg>
    </span>
  </div>
</div>
`;

exports[`2- should render null 1`] = `null`;
